src/list.scss

```diff
@@ -86,8 +87,7 @@ $block: #{$fd-namespace}-list;
 
       .#{$block}__title,
       .#{$block}__secondary,
-      .#{$block}__icon,
-      & {
+      .#{$block}__icon & {
```

Comment:
Could you explain the reason for this change?

Reply:
vscode formatter suggestion, Reverted it back.